src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -773,7 +773,7 @@ private long newestVersion(DHTKey key) {
                             // duplicate the result so that each
                             // waiter gets its own
                             if (debugWaitFor) {
-                                System.out.printf("Triggering waiter for %s\n", pendingWaitFor.getOpUUID());
```

Comment:
debugWaitFor is off by default. Make Log.warningAsyncf so that the debugWaitFor flag has an effect